packages/lambda/src/shared/validate-outname.ts

```diff
@@ -18,13 +20,21 @@ const validateS3Key = (s3Key: string) => {
 };
 
 export const validateOutname = (
-	outName: OutNameInputWithoutCredentials | undefined | null
+	outName: OutNameInputWithoutCredentials | undefined | null,
+	codec?: Codec
 ) => {
 	if (typeof outName === 'undefined' || outName === null) {
 		return;
 	}
 
 	if (typeof outName === 'string') {
```

Comment:
Confusing, but there are download name (name with which file gets downloaded to computer) and outname (name with which file gets saved to S3 bucket). The following command `pnpm exec remotion lambda still testbed react-svg --out-name=still.jpg` will render a PNG still 🤔

Reply:
Fixed that!